src/components/ShellView/ResultGraph.vue

```diff
@@ -982,8 +982,15 @@ export default {
       this.$nextTick(() => {
         if (this.g6Graph) {
           const width = this.$refs.graph.offsetWidth;
-          this.g6Graph.resize(width, parseInt(this.containerHeight));
-          this.g6Graph.fitCenter();
+    
+          // Set graph size based on sidebar state
+          if (this.isSidePanelOpen) {
+            this.g6Graph.setSize(width - this.sidebarWidth, parseInt(this.containerHeight));
+          } else {
+            this.g6Graph.setSize(width, parseInt(this.containerHeight));
+            this.g6Graph.fitCenter();
```

Comment:
Actually I don't think we need to fitCenter automatically. Just ensure the sizing is correct. The user can manually fitCenter.

Automatically do it can be slow.